scripts/smooth_cal_run.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python2.7
+"""Command-line drive script for hera_cal.smooth_cal"""
```

Comment:
It would be nice to add some more documentation here about what is happening. As @nkern pointed out, there are a few steps going on, and it would be nice to give an overview of the steps going on.